feeder/parsers/ENTSOE.py

```diff
@@ -72,11 +72,13 @@ def query_consumption(domain, session):
         'outBiddingZone_Domain': domain,
         'periodStart': now.replace(hours=-24).format('YYYYMMDDHH00'),
         'periodEnd': now.replace(hours=+24).format('YYYYMMDDHH00'),
-        'securityToken': os.environ['ENTSOE_TOKEN']
+        'securityToken': os.environ.get('ENTSOE_TOKEN')
```

Comment:
Mmh ok I actually learned something tonight :p
However, I still think the code should crash here if no key is provided, rather than silently trying an HTTP request that will fail.
Also, rising a proper error before we create the `params` dict is probably the way to go.